tests/GuPaymentTest.php

```diff
@@ -763,6 +763,37 @@ public function testCanRetrieveSoftDeletedUser()
         $this->assertInstanceOf(User::class, $subscription->user);
     }
 
+    public function testDuplicateInvoice(){
```

Comment:
Descer a chaves para seguir o padrão do pacote.
```suggestion
    public function testDuplicateInvoice()
    {
```